sotodlib/io/bookbinder.py

```diff
@@ -515,7 +518,8 @@ def bind(self, outdir, times, frame_idxs, file_idxs, pbar=False, ancil=None,
             oframe['flag_smurfgaps'] = core.G3VectorBool(~filled)
 
             ts = core.G3VectorTime(ts * core.G3Units.s)
-            oframe['signal'] = so3g.G3SuperTimestream(self.readout_ids, ts, data)
+            oframe['signal'] = so3g.G3SuperTimestream(self.readout_ids[res_chans], ts, data[res_chans, :])
+            oframe['non_res_signal'] = so3g.G3SuperTimestream(self.readout_ids[~res_chans], ts, data[~res_chans, :])
```

Comment:
I am not opposed to changing the name if we figure out a better one, but I think "tones" or even "fixed_tones" is misleading, since it is not exclusively fixed tones. Just anything thats not in the tunefile.

Reply:
"untracked"?

Reply:
ooo ya that works